### PR TITLE
the card smoothed on hover

### PR DESCRIPTION
**Pull request by roman-zherebetskyi**
[DEMO LINK](https://roman-zherebetskyi.github.io/layout_catalog_hovers/)

### CODE REVIEW

File: src/styles/_header.scss [hunk 1]
```diff
@@ -37,5 +37,11 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: color 300ms;
```
Comment (igor-1987) [inline src/styles/_header.scss:40]: It would be better to add duration to variables.

File: src/styles/_catalog.scss [hunk 2]
```diff
@@ -14,9 +14,16 @@
...
+
+    &:not(:nth-child(4n)) {
+      margin-right: 48px;
+    }
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+    &:hover {
+      transform: scale(1.2);
+      color: #34568b;
```
Comment (igor-1987) [inline src/styles/_catalog.scss:25]: You can variables with colors.

File: src/styles/_catalog.scss [hunk 3]
```diff
@@ -14,9 +14,16 @@
...
-  }
+    transition: transform 300ms, color 300ms;
+
+    &:not(:nth-child(4n)) {
+      margin-right: 48px;
+    }
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+    &:hover {
```
Comment (igor-1987) [inline src/styles/_catalog.scss:23]: You can add "cursor: pointer", it looks nice when you hover over the card more interactive.

**Review by igor-1987 — COMMENTED** [review]
You can do some changes to improve your code, but in general it's good.

File: src/styles/_catalog.scss (hunk 3 above)
Comment (roman-zherebetskyi) [reply src/styles/_catalog.scss:23]: yes you are right! thank you for suggestion!

File: src/styles/_catalog.scss (hunk 2 above)
Comment (roman-zherebetskyi) [reply src/styles/_catalog.scss:25]: yes I know but I use this color only once in this task

**Review by MaksimusCRS — APPROVED** [review]
GJ

**Pull request closed without merging**